photo/package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "users",
```

Comment:
Same as below - do these have dependencies of their own? You shouldn't need to update package-lock.json if specific services are addressed by specific dependencies anyways.

Reply:
Same as below